addon/components/mdc-linear-progress.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
+import layout from '../templates/components/mdc-linear-progress';
+import { MDCLinearProgressFoundation } from '@material/linear-progress';
+import { MDCComponent } from '../mixins/mdc-component';
+
+const { get, set } = Ember;
+
+const { cssClasses, strings } = MDCLinearProgressFoundation;
+
+export default Ember.Component.extend(MDCComponent, {
+  //region Attributes
+  role: 'progressbar',
```

Comment:
Is it desired that users of `{{mdc-linear-progress}}` can pass in `role` to change it via the HBS? If not this might be better off in `//region Properties`.